packages/twenty-server/src/modules/messaging/message-import-manager/drivers/gmail/utils/sanitize-string.util.ts

```diff
@@ -1,3 +1,32 @@
 export const sanitizeString = (str: string) => {
   return str.replace(/\0/g, '');
 };
+
+/**
+ * Sanitizes an email address to ensure it's in a valid format
+ * - Removes null characters
+ * - Normalizes Unicode characters
+ * - Ensures proper encoding
+ */
+export const sanitizeEmailAddress = (email: string): string => {
```

Comment:
this should be part of the other safeParse function, I don't see why we have two functions